packages/bahmni-services/src/conditionService/conditionService.ts

```
@@ -67,17 +66,76 @@ export async function getConditionPage(
+        {
+          system: FHIR_ENCOUNTER_TAG_SYSTEM,
+          code: 'encounter',
+          display: 'Encounter',
+        },
+      ],
+    },
+    type,
+    subject,
+    partOf,
+    participant: practitionerUUID
+      ? [{ individual: { reference: `Practitioner/${practitionerUUID}`, type: 'Practitioner' } }]
```

ESLint

[error] Replace `{·individual:·{·reference:·`Practitioner/${practitionerUUID}`,·type:·'Practitioner'·}·}` with `⏎··········{⏎············individual:·{⏎··············reference:·`Practitioner/${practitionerUUID}`,⏎··············type:·'Practitioner',⏎············},⏎··········},⏎········`

(prettier/prettier)